### ISSUE TITLE
Em and Kathryn Testing

### ISSUE THREAD

**Issue by emily-johnson** [betheluniversity/classifieds#3]

**Comment by kathrynmagnuson** [comment]
Maybe say "Baby/Kids _Clothes_" so we're clear we're not selling children... :) 

Can we make something so that people can search by category/ so that the categories show up on the home page with the descriptions?

**Comment by kathrynmagnuson** [comment]
*sort by category

**Comment by emily-johnson** [comment]
Note: I'm in Chrome

- Should give format for price, if you don't include $ the sorting gets confused
- Categories box should be longer IMO, especially since you can select multiple
- Should tell people that you can select multiple, or make it easier
- Maybe note about the 180 expiration date

**Comment by e-jameson** [comment]
Things I saw looking through the code. 
- [ ] code has a lot of comments that probably aren't needed. `def search_classifieds` is the method that needs comments the most but it seems to have the least 
- [ ] can we remove the paperkit CSS/JS files from the static folder, I don't think they are being used.
- [ ] why both `styles.css` and `stylesheet.css`? Same with `sorting.js` and `sorttable.js`
- [ ] Templates has inconsistent naming conventions. Should be like `blink_template.html`
- [ ] the `db.Models` from `__init__.py` should be in a `models.py` file
- [ ] could `db_utilities.py` be called `controller` or some sort? `classifieds_controller`? 
- [ ] in forms.py, `from config import ADMINS` should be getting that from `app.config`
- [ ] why are `get_homepage`, `view_classified`, etc in `forms.py`?
- [ ] some of the todos are done, I believe.
- [ ] `get_homepage` is kind of hard to follow. What is `entry[0]`, `entry[1]`, `entry[2]`?
- [ ] Looks like a lot of the `to_send +=` are duplicated
- [ ] could `category_list` be pulled into its own file like we do for Programs, so Content Specialists can edit them?
- [ ] templates should use `url_for` instead of hardcoding paths in <a href=.
- [ ] `app.db` is in Version Control

**Comment by e-jameson** [comment]
And a few general UI/UX things:

- [ ] Buttons on top could be just `Search` instead of `Search through classifieds`. The only one that needs more than 1 word is `View your posted classifieds`. Maybe `My Classifieds`
- [ ] The buttons at https://classifieds.xp.bethel.edu/view-posted/active have inconsistent wording. Some have your, some don't
- [ ] Category field on search and submit are pretty short.
- [ ] Not obvious my name in the top-right is a drop down
- [ ] Homepage is kind of hard to parse, IMO. Maybe it shouldn't be Tabular, and the Title should be more prominent? 

Overall looks really good! Great job @tim-heck @k-larsen

**Comment by kathrynmagnuson** [comment]
![fullsizerender](https://cloud.githubusercontent.com/assets/14840641/15719961/ed95cd20-27f7-11e6-8b7f-de40fa9f8f16.jpg)

**Comment by emily-johnson** [comment]
![screen shot 2016-05-31 at 9 28 47 am](https://cloud.githubusercontent.com/assets/13204220/15719994/0feddc1e-27f8-11e6-8e77-0e794e15e86b.png)

**Comment by kathrynmagnuson** [comment]
on an iPhone, the last option isn't fully on the blue background (sorry for the huge picture) :)

**Comment by c-schwarze** [comment]
Figured I should also add a list :)

- [ ] "Sort Classifieds" shouldn't be an option to choose in the selects
